Jenkinsfile

```diff
@@ -30,6 +38,11 @@ elifePipeline {
 
         stage 'Approval', {
             elifeGitMoveToBranch commit, 'approved'
+            node('containers-jenkins-plugin') {
+                def image = new DockerImage(steps, "xpub/xpub-elife", commit)
+                image.pull()
```

Comment:
pulls the `xpub/xpub-elife:$commit` image and pushes it as `xpub/xpub-elife:approved`